enchant/checker/CmdLineChecker.py

```diff
@@ -42,9 +42,49 @@
 from enchant.checker import SpellChecker
 from enchant.utils import printf
 
+# Helpers
+
+colors = {
+    'normal'         : "\x1b[0m",
+    'black'          : "\x1b[30m",
+    'red'            : "\x1b[31m",
+    'green'          : "\x1b[32m",
+    'yellow'         : "\x1b[33m",
+    'blue'           : "\x1b[34m",
+    'purple'         : "\x1b[35m",
+    'cyan'           : "\x1b[36m",
+    'grey'           : "\x1b[90m",
+    'gray'           : "\x1b[90m",
+    'bold'           : "\x1b[1m"
+}
+
+def color(string, color='normal', prefix=''):
+    """
+    Change text color for the Linux terminal.
+
+    Args:
+        string (str): String to colorify
+        color (str): Color to colorify the string in the following list:
+            black, red, green, yellow, blue, purple, cyan, gr[ae]y
+        prefix (str): Prefix to add to string (ex: Beginning of line graphics)
+    """
+    return colors[color] + prefix + string + colors['normal']
```

Comment:
A nice addition to this function might to be check whether we're printing to a tty, and only add the colors if so.  Something like:

```
    if not sys.stdout.isatty():
        return prefix + string
    [... the bit that does the colouring ...]
```